docs/iidm/model/phaseTapChanger.md

```diff
@@ -51,6 +54,7 @@ twoWindingsTransformer.newPhaseTapChanger()
     .setRegulating(true)
```

Comment:
It seems that in my tests, I have to set the target deadband, the regulation mode, the regulation value before the ````.setRegulating(true)````. I have not tested with steps.

Reply:
In adders, the order does not matter.

Reply:
@annetill is right there is a bug in the adder implementation we need to define the deadband before the regulating to true which is not conform to "builder pattern"